examples/gallery/plot/logo.py

```diff
@@ -0,0 +1,21 @@
+"""
+Logo
+----
+
+The :meth:`pygmt.Figure.logo` method allows to place the GMT logo on a map.  
+
+For more advanced options, see the full option list at :gmt-docs:`logo.html`.
```

Comment:
Users should read the Figure.logo() documentation first, and there is a link to the GMT official documentation in the Figure.logo() documentation.
```suggestion
```